test/system/groups/samples_test.rb

```diff
@@ -820,14 +819,13 @@ def retrieve_puids
       click_button I18n.t('shared.samples.actions_dropdown.import_metadata')
       within('#dialog') do
         attach_file 'file_import[file]', Rails.root.join('test/fixtures/files/metadata/valid_with_puid.csv')
-        find('#file_import_sample_id_column', wait: 1).find("option[value='sample_puid']").select_option
-        within "ul[id='available']" do
+        within "ul[id='Available']" do
```

Comment:
Can you update this id and the one for `Selected` below to use translations as that's how we pass them into the two lists js controller. This comment applies to other locations within this file and other test files for which changes were made to in the PR